mono/metadata/class-accessors.c

```diff
@@ -512,6 +514,11 @@ mono_class_set_nonblittable (MonoClass *klass) {
 	mono_loader_unlock ();
 }
 
+void
+mono_class_contextbound_bit_offset (int* byte_offset_out, guint8* mask_out) {
```

Comment:
This is not a maintainable hack.  We should eliminate those bitfields we need to address.